src/Backend/Modules/MediaLibrary/Layout/Templates/MediaFolderTree.html.twig

```diff
@@ -6,7 +6,12 @@
     </button>
     <div class="form-group">
       <label for="pagetree-search" class="sr-only">{{ 'lbl.search'|trans|ucfirst }}</label>
-      <input id="pagetree-search" name="pagetree-search" value="" maxlength="255" type="text" class="form-control form-control-sm fork-form-text js-tree-search" placeholder="{{ 'lbl.Search'|trans|ucfirst }} {{ 'lbl.MediaFolder'|trans }}">
+      <div class="input-group input-group-sm">
```

Comment:
why are there media library changes in here?

Reply:
@Katrienvh ?

Reply:
This is the tree view from the media library, I made the same changes then in the pages tree